### PR TITLE
Bug Fix - No Config Items Alert

### LINKED ISSUE #265: [Bug] - Config items alert always visible
### Pre-requisites:
- [x] I looked through both [open and closed issues](../issues?utf8=✓&q=is%3Aissue) and did not find an existing bug.

## Description
The alert for no config items is always there

## Expected Behavior
No alert

## Actual Behavior
Shows an alert saying there are no items

## Steps to Reproduce
Have config items and visit admin page

## Other Information
image

### PR DESCRIPTION
**Pull request by tristengeven**
## Pre-Requisites
- [X] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [X] Yes, I included and/or modified tests to cover relevant code **OR** my change is non-technical
- [X] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

## Description of Changes

The "No Config Items" alert was referring to the results object. The reference was updated to the appropriate configItems object.

## Related Issues

Closes #265 - A bug in the dashboard ui

`Co-authored-by: Tatum Hutton <78566681+tatumwashere@users.noreply.github.com>`

### CODE REVIEW

**Comment by tristengeven** [comment]
I meant to open this as a draft pull request.

**Comment by SpencerKaiser** [comment]
@tristengeven is this working as expected now? You only need to open it as a draft if it **is** a draft 😉 if this is ready for us to review/merge, let us know and we'll get it merged!

File: Authors.md [hunk 1]
```diff
@@ -11,3 +11,4 @@
 - Maha Siddique (mxs151731@utdallas.edu)
 - Dhruv Tailor (dhruv6198@gmail.com)
 - Joseph Nalepka (jwn170030@utdallas.edu)
+- Tristen Even (tristengeven@gmail.com)
```
Comment (SpencerKaiser) [inline Authors.md:14]:
```suggestion
- Tristen Even (tristengeven@gmail.com)
- Tatum Hutton (tatumhutton@gmail.com)
```

**Review by SpencerKaiser — APPROVED** [review]
Looks great!

**Pull request merged**